src/python/grapl-common/grapl_common/retry.py

```diff
@@ -1,6 +1,6 @@
 import time
```

Comment:
I was going to use `retry` - ended up not doing so - but moving it to `-common` and giving it better typing is a worthy addition